example/burrow_integration/integration_example.py

```diff
@@ -0,0 +1,217 @@
+    bytes_object = bytes.fromhex(result)
+    ascii_string = bytes_object.decode("ASCII")
+    print(ascii_string)
+
+
+@trace
+def add_asset(address):
+    params = get_first_four_bytes_of_keccak(b"addAsset(string,string)")
+    no_of_param = 2
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding("coin#domain")  # asset id
+    params = params + argument_encoding("500")  # amount of asset
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+
+@trace
+def create_account(address):
+    params = get_first_four_bytes_of_keccak(b"createAccount(string,string,string)")
+    no_of_param = 3
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding("test4")  # source account id
+    params = params + argument_encoding("test")  # domain id
+    params = params + argument_encoding(test_public_key)  #  key
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+
+
+@trace
+def transfer(address):
+    params = get_first_four_bytes_of_keccak(
+        b"transferAsset(string,string,string,string)"
+    )
+    no_of_param = 4
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding(ADMIN_ACCOUNT_ID)  # source account
+    params = params + argument_encoding("userone@domain")  # destination account
+    params = params + argument_encoding("coin#domain")  # asset id
+    params = params + argument_encoding("100")  # amount of asset
+    tx = iroha.transaction(
+        [
+            iroha.command(
+                "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params
+            )
+        ]
+    )
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response = net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+
+@trace
+def balance(address):
+    params = get_first_four_bytes_of_keccak(b"queryBalance(string,string)")
+    no_of_param = 2
+    for x in range(no_of_param):
+        params = params + left_padded_address_of_param(x, no_of_param)
+    params = params + argument_encoding(ADMIN_ACCOUNT_ID)  # account id
+    params = params + argument_encoding("coin#domain")  # asset id
```

Comment:
I suggest to have global constant with domain

Reply:
It is not used so often, and mostly used in different forms.